internal/plugin/resource_vault_start_test.go

```diff
@@ -68,6 +68,48 @@ func TestVaultStartConfigOptionalAttrs(t *testing.T) {
 		"usage_gauge_period":        "30m",
 		"maximum_gague_cardinality": 100,
 	})
+	vaultCfg.UI.Set(true)
+	vaultCfg.CacheSize.Set(42)
+	vaultCfg.DisableCache.Set(true)
+	vaultCfg.DisableMlock.Set(true)
+	vaultCfg.PluginDirectory.Set("/opt/plugins")
+	vaultCfg.PluginTmpdir.Set("/tmp/plugins")
+	vaultCfg.PluginFileUID.Set(1001)
+	vaultCfg.PluginFilePermissions.Set(0644)
```

Comment:
We'll need to use [octal notation](https://go.dev/ref/spec#octal_digits) to please the linters